src/panels/Menu/TopMenuBar/Menus/ViewMenu.tsx

```diff
@@ -59,7 +60,14 @@ export function ViewMenu({ visibleMenuItems, setVisibleMenuItems }: Props) {
 
       <Menu.Item leftSection={<SaveIcon />}>Load/Save Layout</Menu.Item>
       <Menu.Divider />
-      <Menu.Item leftSection={<VisibilityIcon />}>User Visibility</Menu.Item>
+      <Menu.Label>
+        <Group>
+          <VisibilityIcon /> User Visibility
+        </Group>
+      </Menu.Label>
```

Comment:
Looks very busy with the eye icon. How about: 
```suggestion
      <Menu.Label>User Level</Menu.Label>
```
![image](https://github.com/user-attachments/assets/2fbd7143-d8ad-4bbc-8539-d5c2f7bcac85)

or even remove the label completely: 
```suggestion
```
![image](https://github.com/user-attachments/assets/c4c1855f-efc8-4982-ab52-3f1f2e32fb18)


If we want to apply it to other places than just properties it should be renamed anyways, so IMO the property label is probably sufficient